src/views/open-api-docs-view/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './find-latest-stable-version'
+export * from './parse-and-validate-schema'
+export * from './sort-date-version-data'
```

Comment:
How do you feel about explicit exports in this "barrel export" file, like:

```diff
- export * from './find-latest-stable-version'
+ export { findLatestStableVersion } from './find-latest-stable-version'
```

IME, while the `export *` works, the moment you have to debug something, the ambiguity behind it quickly becomes a headache / feels like a footgun

Reply:
ah nice call! I've updated to explicitly named `export { thing }` in this file, and will do this going forward 👍